cmd/kubebuilder/v1/api.go

```diff
@@ -50,10 +50,24 @@ func (o *apiOptions) RunAddAPI() {
 		fmt.Println("Create Resource under pkg/apis [y/n]?")
 		o.doResource = util.Yesno(reader)
 	}
+	if o.doResource {
+		if o.r.Group == "" {
+			log.Fatalf("Must specify --group")
+		}
+		if o.r.Version == "" {
+			log.Fatalf("Must specify --version")
+		}
```

Comment:
group/version is required even if you are scaffolding *just* controller. Can you pl. verify that ? if yes, then we can combine all the validation in common place above ?

Reply:
You are correct.